docs/clear-containers-3.0-quick-install.md

```diff
@@ -0,0 +1,158 @@
+
+### Requirements
+
+  * [go 1.8.3](https://golang.org/)
+  * [glibc-static](https://www.gnu.org/software/libc/libc.html)
+
+### Clear Containers 3.0 components
+
+  * [Runtime](https://github.com/clearcontainers/runtime)
+  * [Proxy](https://github.com/clearcontainers/proxy)
+  * [Shim](https://github.com/clearcontainers/shim)
+  * [Virtcontainers](https://github.com/containers/virtcontainers) (**pause** binary only)
+
+**IMPORTANT:** Do not combine [Clear Containers 2.1](https://github.com/01org/cc-oci-runtime) and [Clear Containers 3.0](https://github.com/clearcontainers) Both projects ship ``cc-proxy`` and they are not compatible with each other.
+
+## Set Environment
+
+### Define GOPATH
+
+```bash
+$ export GOPATH=$HOME/go
+```
+
+### Create GOPATH Directory
+
+```bash
+$ mkdir -p $GOPATH
+```
+
+### Get the code
+
+```bash
+$ go get github.com/clearcontainers/runtime
+$ go get github.com/clearcontainers/proxy
+$ git clone https://github.com/clearcontainers/shim $GOPATH/src/github.com/clearcontainers
+```
+
+## Build and Install Components
+
+### Proxy
+
+```bash
+$ cd $GOPATH/src/github.com/clearcontainers/proxy
+$ make
+     GOBUILD  cc-proxy
+     GEN      cc-proxy.service
+     GEN      cc-proxy.socket
+$ sudo make install
+install -D cc-proxy /usr/libexec/clear-containers/cc-proxy || exit 1;
+install -D -m 644 cc-proxy.service /usr/lib/systemd/system/cc-proxy.service || exit 1;
+install -D -m 644 cc-proxy.socket /usr/lib/systemd/system/cc-proxy.socket || exit 1;
+```
+
+### Shim
+
+```bash
+$ cd $GOPATH/src/github.com/clearcontainers/shim
+$ ./autogen.sh
+$ make
+(CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/sh /home/foo/go/shim/missing autoheader)
+rm -f stamp-h1
+touch config.h.in
+cd . && /bin/sh ./config.status config.h
+config.status: creating config.h
+config.status: config.h is unchanged
+make  all-am
+make[1]: Entering directory '/home/foo/go/shim'
+  CC       src/cc_shim-shim.o
+  CC       src/cc_shim-utils.o
+  CC       src/cc_shim-log.o
+  CCLD     cc-shim
+$ sudo make install
+make[1]: Entering directory '/home/foo/go/shim'
+ /usr/bin/mkdir -p '/usr/libexec/clear-containers'
+  /bin/sh ./libtool   --mode=install /usr/bin/install -c cc-shim '/usr/libexec/clear-containers'
+libtool: install: /usr/bin/install -c cc-shim /usr/libexec/clear-containers/cc-shim
+make[1]: Nothing to be done for 'install-data-am'.
+make[1]: Leaving directory '/home/foo/go/shim'
+```
+
+### Runtime
+
+```bash
+$ cd $GOPATH/src/github.com/clearcontainers/runtime
+$ make
+go build -i -ldflags "-X main.commit="ce08f6bc8b73422e94fd9b9450633799680fbfe0" -X main.version=3.0.0-alpha.0" -o cc-runtime .
+$ sudo make install
+install -D cc-runtime /usr/local/bin/cc-runtime
+install -D config/configuration.toml /usr/local/etc/clear-containers/configuration.toml
+```
+
+### Virtcontainers (pause binary only)
+
+This step will only install the **pause** binary included in [https://github.com/containers/virtcontainers](https://github.com/containers/virtcontainers)
```

Comment:
Maybe something like the following:

    The `pause` binary is required to allow an "empty" pod to be created. Such a pod does not contain any containers: it simply provides the environment to allow them to be created.

Does this sound ok @sboeuf ?